docs/hardware/1-home.md

```diff
@@ -15,6 +15,8 @@ Golioth offers two device SDKs that support a wide range of hardware. Here is
 some guidance on what to use when setting up a development environment for your
 hardware.
 
+### Official Platform Support
```

Comment:
I'm not sure if "Official" is the right counterpart to "Beta" but I don't have a better suggestion, so not requesting any changes.

Reply:
I thing "Platform Support" is descriptive so I removed "Official".